lib/huginn_acumen_order_agent/concerns/invoice_detail_query_concern.rb

```diff
@@ -41,9 +41,10 @@ def process_invoice_detail_response(raw_data)
         results << mapped_item
       rescue => error
         issue_error(AcumenOrderError.new(
+          500,
           'process_invoice_detail_response',
           'Failed while processing Invoice_Detail record',
-          invoice_details,
+          { invoice_details_id: get_field_value(i, 'Invoice_Detail.id') },
```

Comment:
@Jhalter5Stones `i` doesn't exist. I think you mean `invoice_details`

Also, the `Invoice_Detail` field is not `id` Look at the Acumen query to get the correct field name

Reply:
@mrogers-5s Fixed.